apps/dotcms-ui/src/app/api/services/dot-paginator-es-content/dot-paginator-es-content.service.ts

```diff
@@ -0,0 +1,117 @@
+import { take, map } from 'rxjs/operators';
```

Comment:
This service should not have any pagination state, the state of the pagination should be handled by the component, this should just return the data from the API.